app/screens/cart/authorizedCartScreen.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
```

Comment:
this screen should be in DrawerNavigator:
<img width="479" alt="Screenshot 2022-01-10 at 12 18 06" src="https://user-images.githubusercontent.com/58725560/148742598-a3d16827-9e5a-4b8e-92fb-8f1cf408caed.png">